backend/core/src/jurisdictions/jurisdictions.controller.ts

```diff
@@ -50,12 +50,12 @@ export class JurisdictionsController {
     return mapTo(JurisdictionDto, await this.jurisdictionsService.update(jurisdiction))
   }
 
-  @Get(`:jurisdictionId`)
-  @ApiOperation({ summary: "Get jurisdiction by id", operationId: "retrieve" })
-  async retrieve(@Param("jurisdictionId") jurisdictionId: string): Promise<JurisdictionDto> {
+  @Get(`:jurisdictionName`)
```

Comment:
I would like to keep `GET /jurisidctions/:id` endpoint because it's a REST API convention and most endpoints follow it.  `name` search you need here could be implemented through `list()` endpoint. Please add optional query param to `list and do such filtering there. Frontend then should receive and parse a single element array as a response.

Reply:
@pbn4 ,
It makes sense to keep that endpoint, but I don't like using list for this either, since we're only expecting one to return (name column has been set to unique), so having another route like `jurisdictions/byName/:name` makes sense to me.